packages/storybook/src/nimble/table-column/mapping/table-column-mapping.mdx

```diff
@@ -25,35 +33,51 @@ The <Tag name={tableColumnMappingTag}/> renders specific number, boolean, or str
 <Controls of={tableColumnMappingStories.mappingColumn} />
 <ComponentApisLink />
 
-### Icon Mapping
+### Icon mapping
 
 <Description of={iconMappingStories} />
 <Controls of={iconMappingStories.iconMapping} />
 
-### Spinner Mapping
+### Spinner mapping
 
 <Description of={spinnerMappingStories} />
 <Controls of={spinnerMappingStories.spinnerMapping} />
 
-### Text Mapping
+### Text mapping
 
 <Description of={textMappingStories} />
 <Controls of={textMappingStories.textMapping} />
 
-### Empty Mapping
+### Empty mapping
 
 <Description of={emptyMappingStories} />
 <Controls of={emptyMappingStories.emptyMapping} />
 
 ## Usage
 
-### Best Practices
+### Best practices
+
+-   Provide a mapping for every expected record value. Because grouping is
+    performed on the record value, non-mapped record values can result in
+    multiple groups without group labels.
+    -   To improve grouping behavior of a value that shouldn't be rendered in
+        table cells, use a <Tag name={mappingEmptyTag}/> element to provide text
+        for that value's group row.
+-   Avoid having multiple values that map to the same text because grouping and
+    sorting is done on the record value rather than the mapped text.
+
+#### Choosing a mapping style
 
--   Provide a mapping for every expected record value. Because grouping is performed on the record value, non-mapped record values can result in multiple groups without group labels.
-    -   To improve grouping behavior of a value that shouldn't be rendered in table cells, use a <Tag name={mappingEmptyTag}/> element to provide text for that value's group row.
--   Avoid having multiple values that map to the same text because grouping and sorting is done on the record value rather than the mapped text.
+| Best practice                                                                                                                                                                                  | Example                                      |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| **Icon and text, with a text header** <br/> Default configuration where icons exist for the mapping set                                                                                        | <img src={iconTextTextHeader} />             |
```

Comment:
nit: Can we call this the _preferred_ configuration rather than the _default_ configuration? I think the word "default" is a bit confusing in context of a page with API docs where there is a concept of a default behavior of the component.

```suggestion
| **Icon and text, with a text header** <br/> Preferred configuration where icons exist for the mapping set                                                                                        | <img src={iconTextTextHeader} />             |
```